go/kbnm/install_host

```diff
@@ -16,29 +16,54 @@ realpath() {
   fi
 }
 
```

Comment:
You mean this entire script? I'm all for nuking things if they're not meant to be called anymore :)